treeherder/perfalert/perfalert/__init__.py

```diff
@@ -97,16 +97,25 @@ def add_data(self, push_timestamp, value, **kwargs):
         self.data.append(Datum(push_timestamp, value, **kwargs))
         self.data.sort()
 
-    def analyze_t(self, back_window=12, fore_window=12, t_threshold=7):
+    def analyze_t(self, min_back_window=12, max_back_window=24,
+                  fore_window=12, t_threshold=7):
         # Use T-Tests
         # Analyze test data using T-Tests, comparing data[i-j:i] to data[i:i+k]
-        (j, k) = (back_window, fore_window)
+        (j, k) = (min_back_window, fore_window)
         good_data = []
 
-        num_points = len(self.data) - k + 1
+        num_points = len(self.data) - fore_window + 1
```

Comment:
No, it isn't, it was a byproduct of an earlier, abandoned experiment in using more fore data. I'll take it out.